ocs_samples/basic_samples/SDS/JavaScript/NodeJs/WaveData.js

```diff
@@ -30,13 +30,8 @@ module.exports = {
 
     // creates a new wave event
     NextWave: function(interval, multiplier, order) {
-        now = new Date();
-        midnight = new Date();
-        midnight.setHours(0, 0, 0, 0);
-        totalSecondsDay = (now -
-            midnight);
-        _interval = (interval - midnight);
-        radians = ((totalSecondsDay % _interval) / _interval) * 2 * Math.PI;
+        
+        radians = order * Math.PI/32 +1;
```

Comment:
why do we add 1 here? not a big deal either way.

Reply:
This is necessary for Javascript because the values will come back undefined if they are the default value of 0.